a10_octavia/controller/worker/tasks/vthunder_tasks.py

```diff
@@ -177,15 +178,11 @@ def revert(self, subnet, amphora, lb_count_flavor, flavor_data=None, *args, **kw
             return
 
         if flavor_data:
-            try:
-                deployment = flavor_data.get('deployment')
-                if deployment and 'dsr_type' in deployment:
-                    if deployment['dsr_type'] == "l2dsr_transparent":
-                        for amp in amphora:
-                            self.network_driver.remove_any_source_ip_on_egress(
-                                subnet.network_id, amp)
-            except Exception as e:
-                LOG.exception("Failed to revert AllowL2DSR due to: %s", e)
+            deployment = flavor_data.get('deployment')
```

Comment:
Please confirm why we remove the exception handling here?
- throw exception in revert flow will cause resource in pending state. 
- is this caused by missing patches when branch out? or this is in purposed?

Reply:
This is not purposed.
It is caused due to some missing patches in the parent branch out.
It can be fixed by taking latest pull of the upstream master branch.

Reply:
Added the missing try-except code changes.